doc_user/UserManual.adoc

```diff
@@ -828,6 +828,348 @@ image:image251.png[success]
++
+NOTE: The Destination, Charge Type, Discharge Type and Supplier can be defined by the Administrator. For instance, it is advisable to use dedicated charge and discharge types for inventories, as well as a virtual supplier and a virtual ward that will be used as origin and destination of the inventory rectifications. Ask the Administrator or check the <<settings,Settings>> chapter.
+
+The data shown in the *Inventory row table* area are:
+
+* *CODE*: the code of the phamaceutical.
+* *MEDICAL*: the description of the phamaceutical.
+* *NEW LOT*: indicates if the inventory row is for a new lot or not.
+* *LOT's CODE*: the lot code of the pharmaceutical.
+* *DUE DATE*: the due date of the lot.
+* *THEORETICAL*: the theoretical (according to the records) quantity of the lot of the pharmaceutical.
+* *REAL QUANTITY*: the real (physical count) quantity of the lot of the pharmaceutical.
+* *UNIT COST*: the unit cost of the pharmaceutical
+* *TOTAL COST*: the total cost that represents the real value of the lot.
+
+The *Buttons panel* is used to perform some action:
+
+* *[.underline]##S##ave*: this button is used to save the inventory.
+* *[.underline]##L##ot*: this button is used to link new lot to a new specific product (the inventory row that column *New Lot* as value *N*). to perform that, first select the product/lot in the list and then click on this button.
++
+NOTE: This button is enabled just for the product/lot whose column *New Lot* has value *N*, otherwise the button is disabled.
+
+* *[.underline]##D##elete*: used to remove one or more products/lots from the list.
+* *Clean [.underline]##T##able*: used to remove all the inventory rows.
+* *[.underline]##V##alidate*: used to validate the inventory.
+* *[.underline]##C##onfirm*: used to confirm the inventory.
++
+NOTE: The *Confirm* button is enabled once the inventory is validated, until then the button is disabled. See <<inventory-main-characteristics,Main Characteristics>> chapter.
+
+* *[.underline]##C##lose*: used to return to the *_Inventory Browser_* window.
+
+====== 4.4.2.1.1 Insert Inventory Row By Using TextField
+ 
+To insert an inventory row for a medical, type into the *Add medical* field the code or the description of the pharmaceutical and press ENTER. This will open the _Choose a Medical_ window.
+
+image:image254.png[Choose a medical]
+
+Select the desired pharmaceutical or search the pharmaceutical by code or description using the search field and select the pharmaceutical to highlight it, then click on *[.underline]##S##elect* button to add the pharmaceutical as inventory row or click on *[.underline]##C##ancel* to back to the *_Inventory Browser_* window.
+
+====== 4.4.2.1.2 Insert Inventory Row Product By using SELECT Button
+
+To insert an inventory row for a set of medicals, useful for rotational inventories, click on the SELECT button. This will open the Lot Information window.
```

Comment:
```suggestion
To insert an inventory row for a set of medicals, useful for rotational inventories, click the SELECT button. This will open the *_Lot Information_* window.
```

Reply:
Why are the formatting of all buttons references the same, for example the Select button is usually formatted like:  *[.underline]##S##elect*        That isn't true in this line and other places.  Please check for button formatting consistency.

Reply:
ok, SELECT is not even correct with the new GUI... I will change that